hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -74,11 +74,34 @@ def queries(query_group,
         salt '*' nebula.queries hour verbose=True
         salt '*' nebula.queries hour pillar_key=sec_osqueries
     '''
+    query_data = {}
+
     if query_file is None:
         if salt.utils.is_windows():
             query_file = 'salt://hubblestack_nebula/hubblestack_nebula_win_queries.yaml'
         else:
             query_file = 'salt://hubblestack_nebula/hubblestack_nebula_queries.yaml'
+
+    if not isinstance(query_file, list):
+        query_file = [query_file]
+
+    for fh in query_file:
+        if 'salt://' in fh:
+            fh = __salt__['cp.cache_file'](fh)
+        if fh is None:
+            log.error('Could not find file {0}.'.format(fh))
```

Comment:
`fh` will be `None` here. Can you store the old fh (before cp.cache_file) for logging in this error message?

Reply:
Done :)